### PR TITLE
Renamed ingress ClusterOperator name from 'openshift-ingress-operator' to 'ingress'

### PR DESCRIPTION
**Pull request by pravisankar**
https://coreos.slack.com/archives/CCH60A77E/p1549234657654400

### CODE REVIEW

**Comment by pravisankar** [comment]
@openshift/sig-network-edge PTAL

File: pkg/operator/controller/status.go [hunk 1]
```diff
@@ -21,10 +21,14 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	IngressClusterOperatorName = "ingress"
```
Comment (ironcladlou) [inline pkg/operator/controller/status.go]: I thought the request was to drop the `openshift-` prefix — wouldn't this then become `ingress-operator`?

Comment (Miciah) [reply pkg/operator/controller/status.go]: The referenced Slack message says to drop both.

**Pull request merged**